.github/workflows/report.yml

```diff
@@ -0,0 +1,65 @@
+name: "Test generation of comparison reports"
```

Comment:
Can you add a comment above this which briefly summarizes what this workflow is doing / used for, similar to the other files?